tests/kola/data/commonlib.sh

```diff
@@ -86,3 +86,26 @@ is_service_active() {
     # return actual result
     systemctl is-active "${service}"
 }
+
+# Some functions to support version comparison
+vereq() {
+    [ "$1" == "$2" ]
+}
+
+verlt() {
+    vereq $1 $2 && return 1
+    sorted="$(echo -e "$1\n$2" | sort -V)"
+    [  "$1" == $(echo "$sorted" | head -n 1) ]
```

Comment:
Minor: would read better as

```suggestion
    local lowest="$(echo -e "$1\n$2" | sort -V | head -n 1)"
    [  "$1" == $lowest ]
```